src/components/utils/ErrorAlert.tsx

```diff
@@ -41,7 +41,7 @@ export default class ErrorAlert extends Component<IProps, IState> {
                 variant="error"
                 dismissible={!!this.props.onClose}
                 onClose={this.props.onClose}>
-                <FormattedMessage id={this.props.error.code} />
+                <FormattedMessage id={this.props.error.code || "error.app"} />
```

Comment:
Please make a new error message for this because this behaviour is not normal and could lead to pain in trying to debug what is another error